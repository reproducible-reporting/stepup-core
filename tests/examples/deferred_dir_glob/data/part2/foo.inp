Foo input of part 2
