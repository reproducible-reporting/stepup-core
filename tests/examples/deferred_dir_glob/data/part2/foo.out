Foo output of part 2
